src/BribeInitiative.sol

```diff
@@ -78,6 +78,73 @@ contract BribeInitiative is IInitiative, IBribeInitiative {
         bribeToken.safeTransferFrom(msg.sender, address(this), _bribeTokenAmount);
     }
 
+    function _getValidatedLQTYAllocation(
+        DoubleLinkedList.List storage _list,
+        uint256 _epoch,
+        uint256 _prevAllocationEpoch,
+        string memory _errorMsg
+    ) internal view returns (DoubleLinkedList.Item memory _item) {
+        if (_prevAllocationEpoch == 0) {
+            require(_list.isEmpty(), _errorMsg);
+            // Leaving _item uninitialized, as there's never been an allocation
+        } else {
+            require(_list.contains(_prevAllocationEpoch), _errorMsg);
+            _item = _list.getItem(_prevAllocationEpoch);
+            require(_prevAllocationEpoch <= _epoch && (_item.next > _epoch || _item.next == 0), _errorMsg);
+        }
+    }
+
+    // Returns true if there are no existing LQTY allocations to the initiative, and there's no possibility of
+    // allocating any more because the initiative's been unregistered.
+    function _noPresentOrFutureAllocation() internal view returns (bool) {
+        // Note that the initiative could already be unregisterable but not yet unregistered, which we ignore here.
+        // In that case, the stuck bribes can always be extracted in the next epoch _after_ unregistering the initiative.
+        uint256 latestTotalAllocationEpoch = getMostRecentTotalEpoch();
+        return (
+            latestTotalAllocationEpoch == 0 || totalLQTYAllocationByEpoch.getItem(latestTotalAllocationEpoch).lqty == 0
+        ) && governance.registeredInitiatives(address(this)) == UNREGISTERED_INITIATIVE;
+    }
+
+    /// @inheritdoc IBribeInitiative
+    function redepositBribe(uint256 _originalEpoch, uint256 _prevTotalLQTYAllocationEpoch) external {
+        Bribe memory bribe = bribeByEpoch[_originalEpoch];
+        require(bribe.remainingBoldAmount != 0 || bribe.remainingBribeTokenAmount != 0, "BribeInitiative: no-bribe");
+
+        DoubleLinkedList.Item memory totalLQTYAllocation = _getValidatedLQTYAllocation(
+            totalLQTYAllocationByEpoch,
+            _originalEpoch,
+            _prevTotalLQTYAllocationEpoch,
+            "BribeInitiative: invalid-prev-total-lqty-allocation-epoch"
+        );
+
+        require(totalLQTYAllocation.lqty == 0, "BribeInitiative: total-lqty-allocation-not-zero");
+        assert(bribe.claimedVotes == 0); // Can't possibly have claimed anything
+
+        if (_noPresentOrFutureAllocation()) {
+            if (bribe.remainingBoldAmount != 0) {
+                bold.safeTransfer(msg.sender, bribe.remainingBoldAmount);
```

Comment:
Where are we validating that sender is the owner?

Reply:
We are intentionally letting anyone recover the bribes. Since anyone could have deposited those bribes, it doesn't feel right to only let the owner (who's not known currently, since BribeInitiative isn't Ownable) claim them. I expect the bribes to be extracted as MEV in this case.

If you as a bribe depositor couldn't even be bothered to allocate a single wei of LQTY to prevent the bribes from getting stuck in the first place, then you don't deserve to reclaim them.

Reply:
So this means that in the case of 0 LQTY votes, we're going from "no one can claim/redeposit" to "anyone can redeposit"? And in both cases, the issue is fully mitigated by the bribe depositor allocating 1 wei LQTY, right?

Reply:
In case there are no votes in the original epoch in which the bribes were deposited (hence why the're stuck), _and_ the current epoch has no votes either, _and_ the initiative has been disabled then yes, anyone can claim the stuck bribes.

Normally, they would just be moved to the current epoch, where people would be able to vote to see who gets how much. But this is no longer possible if the initiative has been disabled.